django/core/management/commands/migrate.py

```diff
@@ -54,6 +55,10 @@ def add_arguments(self, parser):
             '--run-syncdb', action='store_true',
             help='Creates tables for apps without migrations.',
         )
+        parser.add_argument(
+            '--plan', action='store_true', dest='plan',
+            help='Shows a list of ordered migration actions that Django will perform.',
```

Comment:
ordered -> the ?

Reply:
just to confirm here. this comment is about the use of the "zero article" vs the "definite article"  ? We want:

`Shows a list of the ordered migration actions that Django will perform.`

Correct?

Reply:
I think "ordered" is also unnecessary (i.e. why would they be out of order ;-) ).